### PR TITLE
ZH-trans: add set-up-placement-policies-federation.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+The Federation control plane can be deployed using `kubefed init`.
+-->
+可以使用 `kubefed init` 部署联合控制平面。
+
+<!--
+After deploying the Federation control plane, you must configure an Admission
+Controller in the Federation API server that enforces placement decisions
+received from the external policy engine.
+-->
+部署联邦控制平面之后，必须在联邦 API 服务器中配置一个允许控制器，该控制器强制执行从外部策略引擎接收到的放置决策。
```
Comment (SataQiu) [inline content/zh/docs/tasks/federation/set-up-placement-policies-federation.md]: Admission Controller  建议翻译为 “准入控制器”，下同。
另外，对于同一个名词的翻译最好前后一致

Comment (xichengliudui) [reply content/zh/docs/tasks/federation/set-up-placement-policies-federation.md]: 中。

**Comment by xichengliudui** [comment]
已修改。

File: content/zh/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+-->
+部署联邦控制平面之后，必须在联邦 API 服务器中配置一个准入控制器，该控制器强制执行从外部策略引擎接收到的放置决策。
+
+
+    kubectl create -f scheduling-policy-admission.yaml
+
+<!--
+Shown below is an example ConfigMap for the Admission Controller:
+-->
+下图是接纳控制器的 ConfigMap 示例：
```
Comment (tengqm) [inline content/zh/docs/tasks/federation/set-up-placement-policies-federation.md]: 下面是准入控制器 ...

File: content/zh/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+  metadata) to make decisions.
+* Administrative policy can be managed centrally.
+* Policies can define simple interfaces (such as the `requires-pci` annotation) to
+  avoid duplicating logic in manifests.
+-->
+
+* 位置策略可以引用联邦资源中的任何字段。
+* 放置策略可以利用外部上下文(例如，集群元数据)来做出决策。
+* 管理策略可以集中管理。
+* 策略可以定义简单的接口(例如 ·requirements -pci` 注释)，以避免在清单中重复逻辑。
```
Comment (tengqm) [inline content/zh/docs/tasks/federation/set-up-placement-policies-federation.md]: 笔误！

File: content/zh/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+
+<!--
+Shown below is the command to deploy a ReplicaSet that *does* match the policy.
+-->
+下面显示的命令用于部署与策略匹配的副本集。
+
+    kubectl --context=federation create -f replicaset-example-policy.yaml
+
+<!--
+Inspect the ReplicaSet to confirm the appropriate annotations have been applied:
```
Comment (tengqm) [inline content/zh/docs/tasks/federation/set-up-placement-policies-federation.md]: annotation 统一译为注解

**Comment by xichengliudui** [comment]
已修改。

File: content/zh/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+* `config.yml` 指定 `调度策略` 准入控制器配置文件的位置。
+* `scheduling-policy-config.yml` 指定与外部策略引擎联系所需的 kubeconfig 文件的位置。
+该文件还可以包含一个 `retryBackoff` 值，该值以毫秒为单位控制初始重试 backoff 延迟。
+* `opa-kubeconfig` 是一个标准的 kubeconfig，包含联系外部策略引擎所需的 URL 和凭证。
+
+<!--
+Edit the Federation API server deployment to enable the `SchedulingPolicy`
+Admission Controller.
+-->
+编辑联邦 API 服务器部署以启用 `调度策略` 准入控制器。
```
Comment (tengqm) [inline content/zh/docs/tasks/federation/set-up-placement-policies-federation.md]: SchedulingPolicy 不要翻译

File: content/zh/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+	kubectl -n federation-system edit deployment federation-apiserver
+
+<!--
+Update the Federation API server command line arguments to enable the Admission
+Controller and mount the ConfigMap into the container. If there's an existing
+`--enable-admission-plugins` flag, append `,SchedulingPolicy` instead of adding
+another line.
+-->
+更新 Federation API 服务器命令行参数以启用准入控制器，
+并将 ConfigMap 挂载到容器中。如果存在现有的 `-enable- admissionplugins` 参数，则追加 `SchedulingPolicy` 而不是添加另一行。
```
Comment (tengqm) [inline content/zh/docs/tasks/federation/set-up-placement-policies-federation.md]: -enable- admissionplugins 中多了一个空格

**Pull request merged**